Pico/Motors/Motors.cpp

```diff
@@ -65,6 +65,22 @@ void Motors::moveForward() {
     motor3.moveBackward();
 };*/
 
+void Motors::moveRight() {
+    stopMotors();
+    motor1.moveForward();
+    motor2.moveBackward();
+    motor3.moveBackward();
+    motor4.moveForward();
+};
+
+void Motors::moveLeft() {
+    stopMotors();
+    motor1.moveBackward();
+    motor2.moveForward();
+    motor3.moveForward();
+    motor4.moveBackward();
+};
+
```

Comment:
I think you should remove all `moveDirection()` functions and instead have a general `Move(angle)` method where you enter the desired angle and calculate the forward kinematics of each motor.

Also, the `stopMotors()` function at the beginning should be used for hard stops, and the internal management of the motors speed could benefit from a speed ramp instead.

Reply:
I do have a method called `moveMotors()` that receives as parameters the angle and PWM speed, should I apply it inside this specific logic? 

In regard to the `stopMotors()` method, would you rather prefer that motor speed is decreased proportional to the stop command?